BUILDING.md

```diff
@@ -187,8 +185,8 @@ Let's suppose you build an image called `containerd/build` from the above Docker
 ```sh
 docker run -it --privileged \
     -v /var/lib/containerd \
-    -v ${GOPATH}/src/github.com/opencontainers/runc:/go/src/github.com/opencontainers/runc \
-    -v ${GOPATH}/src/github.com/containerd/containerd:/go/src/github.com/containerd/containerd \
+    -v runc:/go/src/github.com/opencontainers/runc \
```

Comment:
Why?

Reply:
GOPATH is removed intentionally. For newcomers of Go, they don't need to be aware of GOPATH. So the runc source is cloned via `git`, not `go get`.

Reply:
Needs `docker volume create` somewhere, no?

Reply:
Missing the absolute path... I was thinking about passing the source dir cloned in previous step.

Reply:
fixed.